docs/logging-usage.rst

```diff
@@ -392,70 +392,105 @@ Delete a sink:
 Integration with Python logging module
 ---------------------------------------------
 
-
-It's possible to tie the Python :mod:`logging` module directly into Google Cloud Logging. To use it,
-create a :class:`CloudLoggingHandler <google.cloud.logging.CloudLoggingHandler>` instance from your
-Logging client.
+It's possible to tie the Python :mod:`logging` module directly into Google Stackdriver Logging.
+There are different handler options to accomplish this. To automatically pick the default,
+use :meth:``google.cloud.logging.client.get_default_handler`.
 
 .. code-block:: python
 
     >>> import logging
     >>> import google.cloud.logging # Don't conflict with standard logging
     >>> from google.cloud.logging.handlers import CloudLoggingHandler
     >>> client = google.cloud.logging.Client()
-    >>> handler = CloudLoggingHandler(client)
+    >>> handler = client.get_default_handler()
     >>> cloud_logger = logging.getLogger('cloudLogger')
     >>> cloud_logger.setLevel(logging.INFO) # defaults to WARN
     >>> cloud_logger.addHandler(handler)
     >>> cloud_logger.error('bad news')
 
+It is also possible to attach the handler to the root Python logger, so that for example a plain
+`logging.warn` call would be sent to Cloud Logging, as well as any other loggers created. However,
+you must avoid infinite recursion from the logging calls the client itself makes. A helper
```

Comment:
Why bog the users down in this technicality (infinite recursion)? I also am unclear on where and when such a recursion would occur, which further reduces the value of this phrasing.